pxtblocks/blocklyloader.ts

```diff
@@ -2124,11 +2124,6 @@ namespace pxt.blocks {
                     let blockText = '<xml>' +
                         '<block type="variables_change" gap="' + gap + '">' +
                         Blockly.Variables.generateVariableFieldXmlString(mostRecentVariable) +
-                        '<value name="DELTA">' +
```

Comment:
is this not necessary anymore / coming from generateVariableFieldXmlString? or was it just always an ignored field?

Reply:
always ignored! we had the warning for it suppressed--i actually also re-suppressed it but before i did that, i was trying to fix the things causing the warning so this change is from that pass